hoomd/update/box_resize.py

```diff
@@ -55,17 +63,17 @@ class BoxResize(Updater):
     """
 
     def __init__(self, trigger, box1, box2, variant, filter=All()):
-        params = ParameterDict(box1=OnlyTypes(Box,
-                                              preprocess=box_preprocessing),
-                               box2=OnlyTypes(Box,
-                                              preprocess=box_preprocessing),
+        params = ParameterDict(box1=Box,
+                               box2=Box,
                                variant=Variant,
                                filter=ParticleFilter)
         params['box1'] = box1
         params['box2'] = box2
         params['variant'] = variant
         params['trigger'] = trigger
         params['filter'] = filter
+        for attr in ("box1", "box2"):
+            params._set_special_getset(attr, _box_getter, _box_setter)
```

Comment:
Does this fix #705?

Reply:
No idea but I could look into it.

Reply:
No problem, we can solve in another PR if appropriate. Though it is related in spirit to the changes you mare making in this PR.

Reply:
I have verified that currently this is still a bug.